.github/workflows/pr-quality.yml

```
@@ -0,0 +1,210 @@
+          fi
+          
+          echo "overall_coverage=$OVERALL" >> $GITHUB_OUTPUT
+          echo "branch_coverage=$BRANCHES" >> $GITHUB_OUTPUT
+        env:
+          NODE_OPTIONS: '--experimental-vm-modules'
+
+      - name: Generate enhanced PR comment
+        run: |
+          cat > pr_comment.md << EOF
+          ## 🛡️ Code Quality Report
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)